### ISSUE TITLE
Make samples more visible by renaming 'Project' folder to 'Samples'

### ISSUE THREAD

**Issue by Cheesebaron** [MvvmCross/MvvmCross#3335]
## 🏗 Enhancement Proposal

Samples are hidden in the Projects folder, it would be much more appropriate to call it samples.

Also adding a README.md to link to the other samples repo would be a great help for new users.

**Comment by martijn00** [comment]
It was called Projects originally because we already have the https://github.com/MvvmCross/MvvmCross-Samples repo. I guess the rename would be good for visibility.

**Comment by markuspalme** [comment]
@Cheesebaron @martijn00  Are you interested in a PR?

So far I have identified the following references that need to be updated:

- MvvmCross solution file, obvisously (`MvvmCross.sln`)
- `docs/_documentation/samples/samples.md`
- `docs/_documentation/samples/samples.md`
- `docs/_documentation/platform/android/android-support-library.md`
- `.github/CODEOWNERS` 
- `docs/_documentation/platform/macos/mac-view-presenter.md`
- `docs/_documentation/platform/wpf/wpf-view-presenter.md`
- `docs/_documentation/plugins/ios-sidebar.md`
- `docs/_documentation/platform/tvos/tvos-view-presenter.md`
- `docs/_documentation/platform/ios/ios-view-presenter.md`
- `docs/_documentation/platform/android/android-view-presenter.md`

Anything I'm missing?

**Comment by martijn00** [comment]
@markuspalme I think we need to think a bit more about the why, how and what. Maybe we can start a discussion about it first?

**Comment by Cheesebaron** [comment]
A lot of the time when I see people asking on Slack for samples, they miss the treasure that hides in the Projects folder. They just assume we don't have any cool samples.

**Comment by markuspalme** [comment]
I agree with @Cheesebaron. This proposed change would hopefully alleviate some of the confusion for newcomers.

The split between the main repository and the samples repository IMHO is another source of the confusion but I'm not sure what to do about this other than mentioning it in the READMEs of both repos.

**Comment by saamerm** [comment]
I agree with @Cheesebaron , when I search for samples of a framework, I google "... Samples Github" not "... Projects Github"

**Comment by nickrandolph** [comment]
Whilst I agree, I think the Playground should not be moved - imho this should be used primarily as a testing ground for functionality and not necessarily curated to be a good sample for developers to follow. Having said that I don't feel particularly strongly about this so if there's consensus on moving all the projects, then lets get on and do it.
